packtools/sps/validation/front_articlemeta_issue.py

```diff
@@ -1,6 +1,61 @@
 from ..models.front_articlemeta_issue import ArticleMetaIssue
 
 
+def _is_number(value):
+    # a numeric value for issue cannot start with zero
+    # <issue>4</issue>
+    return value.isnumeric() and not value.startswith('0')
```

Comment:
troque `return value.isnumeric() and not value.startswith('0')` por

```python
return value.isnumeric() and str(int(value)) == value
```

Reply:
OK, @robertatakenaka